letter_classification/generate_from_isaac.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import cv2 as cv
+import os
+
+""" 
+example:
+bounding_box_2d_tight_labels_0000.json: 
```

Comment:
Yeah I was thinking we could use one json file to map all labels and classes when we figure out the order of labels instead of one json file for each fullsize  image. For now, since the labels are consistent for each class, I'm  using the labels themselves as the classes (subtract by 2 since 0 and 1 are not letter labels).  For example, label 0 would be class 'y', and all letter images in class 'y' would be in folder `0`.

Reply:
Ok sounds good